packages/components/forms/src/BaseInput/BaseInput.styles.ts

```diff
@@ -1,100 +1,140 @@
 import { css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
 import type { CSSObject } from '@emotion/serialize';
+import type { Density } from '@contentful/f36-utils';
+import { BaseInputInternalProps } from './types';
 
-const getSizeStyles = ({ size }): CSSObject => {
+type getSizeStylesProps = Pick<BaseInputInternalProps, 'size'> & {
+  density?: Density;
+};
+
+const getSizeStyles = ({ size, density }: getSizeStylesProps): CSSObject => {
+  const isHighDensity = density === 'high';
   if (size === 'small') {
     return {
-      padding: `${tokens.spacing2Xs} ${tokens.spacingXs}`,
-      height: '32px',
-      maxHeight: '32px',
+      padding: tokens.spacingXs,
+      minHeight: isHighDensity ? tokens.spacingL : tokens.spacingXl,
+      maxHeight: isHighDensity ? tokens.spacingL : tokens.spacingXl,
```

Comment:
I can see in Storybook the computed height of the `small` TextInput is `24px` whereas in the design spec it is `28px`. But here, with a height of `24px` (`spacingL`) + a vertical padding of `8px` (`spacingXs`), it should also equal `28px`.

Reply:
Good point I didn't see that the rendered on the design for `small` was `28px`, since all the others had the same height as the minHeight I assumed this was the case, or maybe this is a mistake on the design @damann ?? 🤔

Reply:
Good point regarding the `box-sizing` 😉